examples/erp/genseki/collections/posts.tsx

```diff
@@ -185,21 +185,48 @@ export const options = builder.options(fields, {
       ],
     }
   },
+
   author: async ({ body }) => {
+    const opt = (body as any).__options ?? {}
+    const search: string = opt.search ?? ''
```

Comment:
About the type, see here https://github.com/softnetics/kivotos/blob/cb38e1ffd9de56034c34ec8ca5630f3c02422e67/packages/react/src/core/field.ts#L488-L497

Reply:
`take` would be from `query`